docs/dasharo-menu-docs/overview.md

```diff
@@ -15,5 +15,123 @@ can see checkbox results on Dasharo firmware.
 
 ## Dasharo menu guides
 
-This section is under construction and will contain guides for each Dasharo
-menu feature available in our products.
+The main menu is entered by selecting the Setup application in the
```

Comment:
I would drop the `[Here](/variants/novacustom_nv4x_tgl/compatibility-check-results-ubuntu.md) you
can see checkbox results on Dasharo firmware.` a few lines above while editing this file. It does not fit there, at all.

Reply:
I don't understand. Where do you see `[Here](/variants/novacustom_nv4x_tgl/compatibility-check-results-ubuntu.md)` in this file?

Reply:
Few lines above, in the original file (not your changes).

I just reviewed the thing as a whole, and the checbox stuff struck me as it did not fit there at all.
I understand that this is not what you added or modified here, but I would remove it or move somehtere esle when editing this overview.md file.

Reply:
There is also a link to some video, you may judge how useful it is nowadays in scepe of the current menu layout.

Reply:
Ok I see. Totally agree.

Reply:
Removed.